scripts/amplify-cli

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "${ECHOLOCATOR_DEBUG}" ]]; then
+    set -x
+fi
+
+function usage() {
+    echo -n \
+         "Usage: $(basename "$0")
+
+Run amplify CLI inside container.
+"
+}
+
+if [ "${BASH_SOURCE[0]}" = "${0}" ]
+then
+    if [ "${1:-}" = "--help" ]
+    then
+        usage
+    else
+        docker-compose run --rm --entrypoint amplify taui
```

Comment:
Were you imagining that we would continue to deploy these stacks using the Amplify CLI? Based on the docs it seems like handling multiple environments isn't really supported yet. (There's a [beta version of the CLI that supports it](https://aws-amplify.github.io/docs/cli/multienv?sdk=js), albeit in a weird git-dependent way.)

My first instinct on reading this PR is that in #6 we could strip out the Amplify deployment resources created here and manage them via Terraform instead (see the heading "Existing AWS resources" in the [Getting started guide](https://aws-amplify.github.io/docs/js/start#step-5-host-your-app) for an example of how to configure this on the frontend). That would let us stand up dev/staging/production environments and manage them in our usual way. The additional overhead to copy all of these resources over to Terraform isn't very appealing, however, so if you have alternative ideas I'd be interested to hear them.

Reply:
I was wondering as well whether it would be easier to manage via the existing Terraform deployment, or leave these resources to be managed by this Amplify CLI. Another option might be to add static site hosting to the CLI configuration, which is an option, and remove Terraform use from the project, although the multi-environment setup might complicate that, and I'm not sure if that would cover all the resources we expect to use.

These CLI-generated CloudFormation resources should at least be a decent starting point for figuring out what should go in Terraform, if we go that route.

Something I forgot to mention in the issue writeup is that there are a few Cognito settings that weren't captured in the current options, but can be manually configured in the Cognito AWS interface. In particular, the default password rules are pretty restrictive. Here's the defaults:
![cognito_user_pool_default_policies](https://user-images.githubusercontent.com/960264/52504650-17cf9380-2bb7-11e9-94fc-ce250a4b036f.png)

But in general, the Amplify-generated configuration seems acceptable.

Also, something that was an Amplify configuration option we'll probably want to change later: for now, I set it up to allow people to create their own accounts, to ease testing and development, but we'll probably want to turn that off in production and either import or manually create accounts in AWS.

Reply:
Thanks for leaving these notes! Whatever route we take, this context will be useful for making sure that the changes get persisted.